src/pyhf/optimize/common.py

```diff
@@ -3,6 +3,17 @@
 from ..tensor.common import _TensorViewer
 
 
+def _make_stitch_pars(tv=None, fixed_values=None):
```

Comment:
@kratsg Can we get a short docstring here so that it is more evident in the future why this exists?